domain/model/state/modelstate.go

```diff
@@ -105,7 +106,28 @@ func (s *ModelState) Delete(ctx context.Context, uuid coremodel.UUID) error {
 	return nil
 }
 
-// GetModel returns model information that has been set in the
+// GetModelConstraints returns the currently set constraints for the model.
+// The following error types can be expected:
+// - [modelerrors.NotFound]: when no model exists to set constraints for.
+func (s *ModelState) GetModelConstraints(ctx context.Context) (constraints.Value, error) {
+	// TODO: Implement this method.
+	return constraints.Value{}, nil
+}
+
+// SetModelConstraints sets the model constraints to the new values supplied
+// overriding and previously set values.
```

Comment:
This is the problem with copying interface method comments. If something changes, like typo fixes, you need to somehow hunt down all the places to change, and it's error prone to know if you've got them all. I know others disagree - my preference is to document on the interface and refer to that on the implementation method - it's one click away with any decent IDE, and in fact not even a click as you can hover other the method and the doc pops up.

So we have significant text here and below so we have better be sure to always check the other places to ensure we are consistent. This PR needs some tweaks so will have changes in a couple of places.

Reply:
Fixed up. I don't have a dog in this fight.